### ISSUE TITLE
[BUG REPORT] - release 0.3 now

### ISSUE THREAD

**Issue by sabaasa** [Atlas-OS/Atlas#947]
### Description

release 0.3
nownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownownowi have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrheai have diarrhea

### Steps to reproduce (add screenshots if applicable)

1. taco
2. toilet
3. diarrhea

### Expected behavior

i expected my shit to not be liquid

### Actual behavior (add screenshots if applicable)

my shit was liquid

### Atlas Version

Atlas 10 22H2

### Desktop information

no

### Requisites

- [X] This is not a support issue or a question. For any support, questions or help, join our [Discord server](https://discord.atlasos.net).
- [X] I performed a [cursory search of the issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I checked the [documentation](https://docs.atlasos.net) to understand that the issue I am reporting is not normal behavior.
- [X] I understand that not filling out this template correctly will lead to the issue being closed.

### Additional content

![image](https://github.com/Atlas-OS/Atlas/assets/75037904/44a57e22-d4af-4011-ad4a-0190c4029622)

**Comment by JayXTQ** [comment]
lyssa, i will steal your toes

**Comment by JayXTQ** [comment]
sobs

**Comment by Totallynotmwa** [comment]
8:30 (before school): Watching Skibidi Toilet with friends
9am: Little dark age edit making class
10am: Rizzonomics Class (Kai Cenat is teacher)
11am: Looksmaxxing class
12am: Lunch (McDonalds grimace shake from Ohio)
1pm: Bussin Gyatt Studies
2pm: Atlas os History
3pm: L Rizz Awareness Seminar
4pm: Baltimore Language class

**Comment by Totallynotmwa** [comment]
I love diarrhea
Yes that is right, I love to have diarrhea. One of my favorite things to do is take a laxative on weekends when I have absolute nothing to do and I know I’ll be at the house all day. Sometimes when I can’t wait the 8 hours a laxative will take I’ll do a salt water flush and I’ll be peeing out my butthole in 30 minutes. I’ve even found for me that 100% grapefruit juice will do the trick if I drink half a gallon in a day. There are so many aspects of it that make it one of the best experiences.
The first aspect I love is the rush. Just going about my business and then all of a sudden I know I might poop my pants. That it is time to get on a toilet right now and nothing else matters. It’s an adrenaline rush when the stomach starts to gurgle and you know you can’t stop what’s about to happen. That I am totally at the mercy of my bowels.
The second is simply the feeling. I love the feeling of relief I get in my stomach, the feeling of it rushing out of my butt hole, and it is one of those good burn feelings in the butt hole too. I love that fiery butt hole burn. Then when you go to wipe it’s kind of like you are scratching that ultimate itch. To put it simply from start to finish the feeling of diarrhea has multiple aspects that all feel good and all of them have their special place in my heart.
The third aspect I love is the smell. It’s like when you have a good fart and it stinks really bad and you kind of sniff it in and you are like dang, I just did that. The smell of diarrhea is like I just opened a sewer line right in my toilet and I love it. It’s just a great scent to sit there and enjoy for a minute. A nasty type of good that makes you feel a little dirty.
The fourth aspect would have to be looking at what came out. Just standing to wipe and looking at what once used to be a perfectly clean toilet is now black. Just black water with black spatter all around the bowl of the toilet. I love to stand there for a second and look at it and say “dang look at all that bad stuff I just expelled from my body.” That’s a great feeling looking at it and knowing how you’ve just purged your body.
To put it simply, I highly recommend diarrhea.

**Comment by Js0nGIT** [comment]
https://tenor.com/view/what-the-flop-floppa-hi-mia-gif-19431339

**Comment by Js0nGIT** [comment]
![what-the-flop-floppa](https://github.com/Atlas-OS/Atlas/assets/83360782/e91c8067-065f-4431-bb68-85038be4a370)

**Comment by JayXTQ** [comment]
locking conversation

**Comment by Js0nGIT** [comment]
sorry cant drink grapefruit any other methods
